newsroom/assets/utils.py

```diff
@@ -0,0 +1,101 @@
+import os
+import bson
+
+from werkzeug.utils import secure_filename
+from motor.motor_asyncio import AsyncIOMotorGridOut
+from typing import Any, Mapping, Optional, Sequence, cast
+
+from superdesk.flask import url_for
+from superdesk.core import get_current_app
+from superdesk.upload import upload_url as _upload_url
+from superdesk.media.media_operations import guess_media_extension
+
+from .module import ASSETS_ENDPOINT_GROUP_NAME, ASSETS_RESOURCE
+
+CACHE_MAX_AGE = 3600 * 24 * 7  # 7 days
+
+
+def get_newsroom_app():
```

Comment:
This would be good to have in the new [newsroom.core](https://github.com/superdesk/newsroom-core/blob/async/newsroom/core/__init__.py) module, so that Newshub specific app features can be imported from there.